cmd/clusterctl/examples/openstack/provider-component/debug/manager/loadbalancer.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Service
```

Comment:
Hmm. Not sure if we want this public reachable. Just in case someone forgets to redeploy the normal version.

Maybe it is enough to document how to port-forward into the pod and connect dlv to `localhost:31000`?

Reply:
I'm not sure if that really changes the problem. You could still forget that you forwarded the port, and be vulnerable. Maybe we can add "Debug" to the resources created in some way so it is clear to developers when pods are being run in debugging mode

Reply:
port-forward just forwards a local port to the port on a pod. It does not expose anything reachable from the outside.